service/api/types.go

```diff
@@ -405,6 +405,8 @@ type DebuggerCommand struct {
 	ReturnInfoLoadConfig *LoadConfig
 	// Expr is the expression argument for a Call command
 	Expr string `json:"expr,omitempty"`
+	// SkipCalls is true if 'step-instruction' commands should step over function calls
```

Comment:
I think maybe this should just be a different value of `Name`. If we decide to leave it this way however `SkipCalls` must be added after `UnsafeCalls` for backwards compatibility reasons.